src/result-list.js

```diff
@@ -12,22 +12,48 @@ function ResultList(retriever, options) {
   this._retriever = retriever;
   this._onFailure = options.onFailure;
   this._onSuccess = options.onSuccess;
-  this._page = options.page || 1;
+  this._page = options.page ? options.page - 1 : 0;
   this._perPage = options.perPage || 20;
+  this._requestPromises = [];
+
+  this._retriever.query({per_page: this._perPage});
 }
 
 ResultList.prototype.next = function() {
-  this._retriever.query({page: this._page, per_page: this._perPage});
   this._page += 1;
+
+  this._retriever.query({page: this._page});
+
+  var promise = this._requestPromises[this._page] ?
+    this._requestPromises[this._page] : this._get();
+
+  return promise;
+};
+
+ResultList.prototype.prev = function() {
```

Comment:
Yes I agree it is a common abbreviation. But that does not mean it can't confused with "preview", "prevent" etc.

Abbreviating a function/variable name to save time typing is silly. Spelling a name out to make it clear will probably save more time. A developer may see `prev` used and not be totally convinced `prev == previous` off the bat, and will want to make sure by reading documentation. Instead we can just use `previous` and they should convinced the purpose of the function.